content/learn/book/input/gamepad/_index.md

```diff
@@ -0,0 +1,13 @@
++++
+title = "Mouse"
```

Comment:
"Mouse" is already used in section 3. I'm assuming this was just a copy/paste error?

Reply:
Yep just a typo. There was a lot of tedium involved in the structure and all the files are called _index.md :(

Reply:
I haven't found a way yet, but id love to be able to flatten out this structure. I also like to be able to include a weight prefix in the name instead of defining it in the file ... that way its automatically sorted in IDEs / filesystems.

Reply:
@mockersf any ideas? I would have loved this.

Reply:
```suggestion
title = "Gamepad"
```